src/server/src/test/java/io/cassandrareaper/acceptance/BasicSteps.java

```diff
@@ -1695,7 +1717,11 @@ private static void createTable(String keyspaceName, String tableName) {
         }
       }
 
-      tmpSession.execute(createTableStmt);
+      try {
+        if (null == tmpSession.getCluster().getMetadata().getKeyspace(keyspaceName).getTable(tableName)) {
+          tmpSession.execute(createTableStmt);
```

Comment:
@michaelsembwever I'm wondering: the usual recommended way when writing integration tests with a remote datastore is to ensure that the datastore is empty at the beginning of each test (a test data should not leak in another test).

Here don't we take the risk to already have a keyspace & table from a previous test (but with a different configuration) and then re-use it? If the configuration is not the same it may yield strange failures.

Of course it's no little change, so maybe a better candidate for another PR. What do you think?

Reply:
i think it's the same test just running the setup code concurrently… but haven't looked into it any further.